engine/src/flutter/shell/platform/windows/flutter_window.cc

```diff
@@ -305,6 +308,18 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+FlutterEngineDisplayId FlutterWindow::GetDisplayId() {
+  FlutterEngineDisplayId const display_id =
+      reinterpret_cast<FlutterEngineDisplayId>(
+          MonitorFromWindow(GetWindowHandle(), MONITOR_DEFAULTTONEAREST));
+  if (auto const display = display_monitor_->FindById(display_id)) {
+    return display->display_id;
+  }
+
+  FML_LOG(ERROR) << "Current monitor not found in display list.";
+  return display_id;
```

Comment:
Can this be simplified?

```suggestion
  if (!display_monitor_->FindById(display_id)) {
    FML_LOG(ERROR) << "Current monitor not found in display list.";
  }

  return display_id;
```